### ISSUE TITLE
AUR?

### ISSUE THREAD

**Issue by Krutonium** [hzeroo/twitch-desktop#3]
I would love to have this in the form of an AUR package for Arch Linux. Is there anyone who would be willing to do so? I would myself but I don't have the time nor patience to do it myself.

**Comment by victorz** [comment]
Yeah would be nice, but is it possible, considering the need for a `client_id` during building?

**Comment by hryuk** [comment]
Twitch API changed a lot. I'm updating the app to the new version, and improving general usage as well. I didn't read a enough about client_id limits, but since streamlink is using a fixed client_id and is used widely, i think it can be possible. 

Anyway, starting with the new version the linux release will be on Appimage format, so the installation will be easier.

**Comment by victorz** [comment]
@Krutonium I use Arch, so if I manage to build the app at all in the future (when #4 is resolved), then maybe I can get an Arch package uploaded to the AUR. :man_shrugging:

**Comment by Krutonium** [comment]
Thanks my man!
